### PR TITLE
Add (U)INT128_C types to core.stdc.stdint.h

### PR DESCRIPTION
**Pull request by cschlote**
Added some missing aliases for symmetrie reasons.

### CODE REVIEW

File: src/core/stdc/stdint.d [hunk 1]
```diff
@@ -512,6 +512,12 @@ alias INT32_C = _typify!int32_t;
 ///
 alias INT64_C = _typify!int64_t;
 
+static if (is(ucent))
```
Comment (RazvanN7) [inline src/core/stdc/stdint.d:515]: This should probably be `cent`

**Review by RazvanN7 — COMMENTED** [review]
I don't really see the point in adding this since recent discussions have highlighted that @WalterBright prefers a library solution. In short, cent/ucent will never get implemented.

**Comment by cschlote** [comment]
Well, C defines uint128_t as a type, and the RISC-V architecture also provides the commands for it. There also exists first samples of hardware implemeting the 128 bit registers. So in the forrseeable future there will be native support for 128 integer arthmetics on assembly level.

For this reason we should also provide the C defines from stdint.h as well as a 128 bit integer type in D. For current architectures this is of course just some run-time support code, which emulates 128 bit with 2x64 or 4x32 bit integers.

**Comment by ibuclaw** [comment]
They will likely be exposed via a special enum, and the keywords cent/ucent are due to be deprecated.

**Pull request closed without merging**